src/modules/home/communityBenefits/communityBenefits.tsx

```diff
@@ -0,0 +1,25 @@
+"use client";
+import Image from "next/image";
+import { Button } from "@/components/ui/buttons/button";
+import arrow from "@public/images/icons/arrow.svg";
+import { communityBenefitsList } from "./helper";
+import { CommunityBenefitCard } from "./CommunityBenefitCard";
+
+export const CommunityBenefits = () => {
+  return (
+    <section className="md:flex-col-2 flex flex-col justify-center gap-16 lg:mx-6">
+      <h1 className="text-center text-2xl font-bold lg:text-left">
+        Our Community Benefits
+      </h1>
+      <section className="grid grid-cols-1 gap-12 sm:grid-cols-2 md:grid-cols-3 md:gap-16 lg:grid-cols-4 lg:gap-24">
+        {communityBenefitsList.map(({ name, imageurl }) => (
+          <CommunityBenefitCard key={name} name={name} imageurl={imageurl} />
+        ))}
```

Comment:
```suggestion
        {communityBenefitsList.map(({ name, imageurl }) => 
          <CommunityBenefitCard key={name} name={name} imageurl={imageurl} />
        )}
```

Reply:
Just wanted to point out if we really want to avoid these types of parenthesis on arrrow functions, let's override on the configs else every-time we save it will automatically be added since it runs prettier/eslint to check formatting and linting as default React conventions. Obviously, it's a team preference and not a big deal, however I would stick to defaults to not override a lot.